app/views/boards/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<body>
+ <div class="container">
+ <table class="block-center"
+   <table class="one">
+     <% 4.times do %>
+      <tr class="even">
+ 	     <% 8.times do %>
+ 	      <td></td>	 
+ 	     <% end %>
+      </tr>
+      <tr class="odd">
+       <% 8.times do %>
+ 	       <td></td>
+       <% end %>
+    	</tr>
+ 	   <% end %>
+    </table>
+   </table>
+  </div>
+ </div>
```

Comment:
There's only one opening div tag at the top but you have two closing div tags at the bottom.